src/main/kotlin/model/Interventions.kt

```diff
@@ -98,8 +98,9 @@ class Interventions private constructor() {
       InterventionTeams.crsProvider.uses(ui, "maintains directory and delivery of dynamic framework interventions and services in")
       ProbationPractitioners.nps.uses(ui, "refers and monitors progress of their service users' interventions and services in")
 
-      service.delivers(InterventionTeams.crsProvider, "emails new referrals", "gov.uk notify")
-      service.delivers(ProbationPractitioners.nps, "emails attendance and safeguarding issues", "gov.uk notify")
+      service.uses(Notify.system, "delivers notifications via")
+      Notify.system.delivers(InterventionTeams.crsProvider, "emails new referrals", "email")
+      Notify.system.delivers(ProbationPractitioners.nps, "emails attendance and safeguarding issues", "email")
```

Comment:
I considered this earlier, but going through a service loses the connection between interventions and the actors

So, practically, finding out which service sends CRS providers emails becomes impossible

I think for us, that service-user link has more value than having notify as an external system

Reply:
Adding `service.uses(Notify.system, "delivers notifications via")` on top of the original would work though

So perhaps this is an abstraction of "intermediary tool", something like
```
Notify.delivers(from = service, to = InterventionTeams.crsProvider, label = "emails new referrals")
```
and then could register both?

Reply:
That sounds sensible - I agree that losing that link would be a real pain and make some very messy diagrams, but in terms of identifying data flows, threat models, etc, notify should be treated as a proper system. Not quite sure how your intermediary tool approach works as I'm pretty new to this repo/structure, but it sounds sensible from what you've said.

Reply:
Sorry, bad choice of words from my part (and lots of delay... sorry)

I meant like a DSL, similar to Outside HMPPS:
```
class OutsideHMPPS {
  companion object : addTo {
    override fun addTo(element: Element) {
      when (element) {
        is SoftwareSystem -> element.setLocation(Location.External)
        is Person -> element.setLocation(Location.External)
      }
      {snip}
```